src/python/pants/backend/native/subsystems/native_toolchain.py

```diff
@@ -0,0 +1,51 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from contextlib import contextmanager
+
+from pants.backend.native.subsystems.clang import Clang
+from pants.backend.native.subsystems.gcc import GCC
+from pants.backend.native.subsystems.platform_specific.linux.binutils import Binutils
+from pants.binaries.execution_environment_mixin import ExecutionEnvironmentMixin
+from pants.subsystem.subsystem import Subsystem
+from pants.util.memo import memoized_property
+from pants.util.osutil import get_os_name, normalize_os_name
+
+
+class NativeToolchain(Subsystem, ExecutionEnvironmentMixin):
+
+  options_scope = 'native-toolchain'
+
+  PLATFORM_SPECIFIC_TOOLCHAINS = {
```

Comment:
Should be a private field most likely?

Reply:
Absolutely, see [the current version](https://github.com/cosmicexplorer/pants/blob/0b1c385cba373b3da4ce2a431f1868c6ede30086/src/python/pants/backend/native/subsystems/native_toolchain.py#L82).